lib/Minz/View.php

```diff
@@ -237,6 +238,37 @@ public static function appendStyle($url, $media = 'all', $cond = false) {
 		);
 	}
 
+	/**
+	 * @param array $themeColors
+	 * @return void
+	 */
+	public static function appendThemeColors($themeColors): void {
+		self::$themeColors = $themeColors;
+	}
+
+	/**
+	 * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color
+	 *
+	 */
+	public static function metaThemeColor(): String {
+		$meta = '';
+
+		if (!empty(self::$themeColors['light'])) {
+			$meta .= '<meta name="theme-color" media="(prefers-color-scheme: light)" content="' . self::$themeColors['light'] . '" />';
+		}
+		if (!empty(self::$themeColors['dark'])) {
+			$meta .= '<meta name="theme-color" media="(prefers-color-scheme: dark)" content="' . self::$themeColors['dark'] . '" />';
+		}
+		if (!empty(self::$themeColors['default'])) {
+			$meta .= '<meta name="theme-color" content="' . self::$themeColors['default'] . '" />';
+		}
+		if (empty(self::$themeColors['default']) && !empty(self::$themeColors && empty(self::$themeColors['light']) && empty(self::$themeColors['dark']))) {
+			$meta .= '<meta name="theme-color" content="' . self::$themeColors . '" />';
```

Comment:
```suggestion
			$meta .= '<meta name="theme-color" content="' . htmlspecialchars(self::$themeColors) . '" />';
```

Same for the other lines. Although the input is not supposed to be anything else than a colour string, as themes can be provided by third-parties, we should have some minimum security.

Reply:
thanks for the review. Done.